thirdparty/libwebp/src/dec/frame_dec.c

```diff
@@ -694,7 +694,7 @@ static int AllocateMemory(VP8Decoder* const dec) {
   const size_t mb_data_size =
       (dec->mt_method_ == 2 ? 2 : 1) * mb_w * sizeof(*dec->mb_data_);
   const size_t cache_height = (16 * num_caches
-                            + kFilterExtraRows[dec->filter_type_]) * 3 / 2;
+                            + kFilterExtraRows[dec->filter_type_]) * 1.5;
```

Comment:
1. We do not ever change things in third-party code.
2. You changed behavior here, this used to be integer division.